### ISSUE TITLE
[Bug] - Desktop versions of websites are being displayed instead of mobile versions

### ISSUE THREAD

**Issue by Walkinstownlad** [mozilla-mobile/fenix#13868]
When I load websites in Firefox for Android it's loading the desktop version of the site instead of the mobile version. I've tried clearing cache & cookies, etc. but the issue remains. Randomly at times the correct mobile version will load properly, but with no consistency whatsoever. The "desktop site" slider is toggled to disabled status & I've disabled my installed add-ons(dark reader & ublock origin) to make sure they're not causing the issue directly. 

This issue only began occurring when I installed Firefox for Android version 79.0 from the stable release channel. 

* Android device: OnePlus 7 Pro (Android 10)
* Fenix version: 79.0.4 (from stable release channel)

**Comment by s-ankur** [comment]
Hi @Walkinstownlad, does this only happen when you leave the app and come back after some time? If so, then it could be related to https://github.com/mozilla-mobile/fenix/issues/10926

**Comment by Walkinstownlad** [comment]
Hey Ankur,Thanks for getting back to me about this. This issue resolved itself once I downloaded & installed stable release version 80.0. Figured it was patched in the update. Either way I'll take it! 🙂On Sep 1, 2020 12:01 PM, Ankur Sonawane <notifications@github.com> wrote:
Hi @Walkinstownlad, does this only happen when you leave the app and come back after some time? If so, then it could be related to #10926
